V4-Upgrade-Guide.md

```diff
@@ -48,3 +49,25 @@ To disable caching set `useCache: false` in the options, for example in `execute
 .execute({ destinationName: 'DESTINATION', jwt: 'JWT', useCache: false })
 ```
 
+## Set `iasToXsuaaTokenExchange` to `true` to enable IAS to XSUAA token exchange
+
+**Token exchange from IAS to XSUAA is now disabled by default. Set `iasToXsuaaTokenExchange` to `true` explicitly if token exchange is expected.** 
+
+This change affects the default behaviour of following functions
+
+- `getDestination()`
+- `getAllDestinationsFromDestinationService()`
+- `registerDestination()`
```

Comment:
[req] I don't believe this function is affected. The options don't allow sending this.

Reply:
True. Overlooked `Pick`.